src/magento2/application/overlay/phpcs.xml

```diff
@@ -59,7 +59,7 @@
 
     <rule ref="SlevomatCodingStandard.TypeHints.DeclareStrictTypes">
         <properties>
-            <property name="newlinesCountBetweenOpenTagAndDeclare" value="2"/>
+            <property name="linesCountBeforeDeclare" value="1"/>
```

Comment:
as this is an overlay, projects would need to upgrade slevomat/coding-standard or revert in an override